core/templates/domain/blog/blog-dashboard-backend-api.service.spec.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for BlogDashboardBackendApiService.
+ */
+
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { BlogDashboardBackendApiService } from 'domain/blog/blog-dashboard-backend-api.service';
+import { BlogDashboardPageConstants } from 'pages/blog-dashboard-page/blog-dashboard-page.constants';
+import { BlogPostSummary } from 'domain/blog/blog-post-summary.model';
+
+describe('Blog Dashboard backend api service', () => {
+  let bdbas: BlogDashboardBackendApiService;
+  let httpTestingController: HttpTestingController;
+  let successHandler = null;
+  let failHandler = null;
```

Comment:
Assignment to null not required here and elsewhere

Reply:
Removed.